src/distributors/zulu-installer.ts

```diff
@@ -76,7 +76,7 @@ export class ZuluDistributor extends JavaBase {
     const javaPath = await tc.cacheDir(
       archivePath,
       this.toolcacheFolderName,
-      javaRelease.resolvedVersion,
+      `${javaRelease.resolvedVersion}`,
```

Comment:
may be `javaRelease.resolvedVersion.toString()`

Reply:
Done. Thank you.